lib/Jsrt/ChakraCore.h

```diff
@@ -292,7 +292,13 @@ CHAKRA_API
 /// <param name="value">JavascriptString value</param>
 /// <param name="buffer">Pointer to buffer</param>
 /// <param name="bufferSize">Buffer size</param>
-/// <param name="written">Total number of characters written</param>
+/// <param name="writtenLength">Total number of characters written. This is only
```

Comment:
Oh, also just noticed in the `remarks` section above it still refers to `written` instead of `actualLength`.